### PR TITLE
Bcolz

### PR DESCRIPTION
**Pull request by manifestoso**
py-gemini dep

### CODE REVIEW

**Comment by manifestoso** [comment]
@adamjstewart It looks like we have got a broken package URL  https://packages.microsoft.com/ubuntu/18.04/prod/dists/bionic/main/binary-amd64/Packages.bz2
A triggered rebuild may be needed once it fixed.

File: var/spack/repos/builtin/packages/py-bcolz/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    """bcolz provides columnar and compressed data containers. Column
+    storage allows for efficiently querying tables with a large number
+    of columns. It also allows for cheap addition and removal of column.
+    In addition, bcolz objects are compressed by default for reducing
+    memory/disk I/O needs. The compression process is carried out internally
+    by Blosc, a high-performance compressor that is optimized for binary data.
+    """
+
+    homepage = "https://github.com/Blosc/bcolz"
+    url      = "https://files.pythonhosted.org/packages/5c/4e/23942de9d5c0fb16f10335fa83e52b431bcb8c0d4a8419c9ac206268c279/bcolz-1.2.1.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-bcolz/package.py]: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#url

File: var/spack/repos/builtin/packages/py-bcolz/package.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    version('1.2.1', sha256='c017d09bb0cb5bbb07f2ae223a3f3638285be3b574cb328e91525b2880300bd1')
+
+    depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-bcolz/package.py]:
```suggestion
    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-bcolz/package.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    depends_on('py-numpy@1.8:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-bcolz/package.py]:
```suggestion
    depends_on('py-numpy@1.7:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-bcolz/package.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-bcolz/package.py]:
```suggestion
    depends_on('py-setuptools@18.1:', type='build')
    depends_on('py-setuptools-scm@1.5.5:', type='build')
    depends_on('py-cython@0.22:', type='build')
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-bcolz/package.py (hunk 2 above)
Comment (manifestoso) [reply var/spack/repos/builtin/packages/py-bcolz/package.py]: @adamjstewart it looks like `python==3.5` is supported as
https://github.com/Blosc/bcolz/tree/1.2.1#Requisites

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-bcolz/package.py]: Yes, Spack version ranges are inclusive, so `2.7:2.8,3.5:` is equivalent to 2.7 and 3.5+.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**